### ISSUE TITLE
multi_gpu_model() corrupts model structure

### ISSUE THREAD

**Issue by raehyuk** [keras-team/keras#12694]
<em>Please make sure that this is a Bug or a Feature Request and provide all applicable information asked by the template.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.</em>  

**System information**  
- Have I written custom code (as opposed to using example directory):  
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  
- TensorFlow backend (yes ):  
- TensorFlow version:  1.13.1
- Keras version:  2.2.4
- Python version:  3.5
- CUDA/cuDNN version:  cuda 9.0  cudnn 7.0.5
- GPU model and memory:  GTX 1080Ti

You can obtain the TensorFlow version with:  
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"  
You can obtain the Keras version with:  
python -c 'import keras as k; print(k.__version__)'  

**Describe the current behavior**  
I have two gtx1080ti in my computer and I tried to use both gpus in parallel with multi_gpu_model().

But when I use multi_gpu_model, it corrupts model. This is the model.summary() of my network.
Layer (type)                 Output Shape              Param #   
=================================================================
conv1 (Conv3D)               (None, 16, 144, 192, 64)  5248      
_________________________________________________________________
pool1 (MaxPooling3D)         (None, 16, 72, 96, 64)    0         
_________________________________________________________________
conv2 (Conv3D)               (None, 16, 72, 96, 128)   221312    
_________________________________________________________________
pool2 (MaxPooling3D)         (None, 8, 36, 48, 128)    0         
_________________________________________________________________
conv3a (Conv3D)              (None, 8, 36, 48, 256)    884992    
_________________________________________________________________
conv3b (Conv3D)              (None, 8, 36, 48, 256)    1769728   
_________________________________________________________________
pool3 (MaxPooling3D)         (None, 4, 18, 24, 256)    0         
_________________________________________________________________
conv4a (Conv3D)              (None, 4, 18, 24, 512)    3539456   
_________________________________________________________________
conv4b (Conv3D)              (None, 4, 18, 24, 512)    7078400   
_________________________________________________________________
pool4 (MaxPooling3D)         (None, 2, 9, 12, 512)     0         
_________________________________________________________________
conv5a (Conv3D)              (None, 2, 9, 12, 512)     7078400   
_________________________________________________________________
conv5b (Conv3D)              (None, 2, 9, 12, 512)     7078400   
_________________________________________________________________
zero_padding3d_1 (ZeroPaddin (None, 2, 11, 14, 512)    0         
_________________________________________________________________
pool5 (MaxPooling3D)         (None, 1, 5, 7, 512)      0         
_________________________________________________________________
flatten_1 (Flatten)          (None, 17920)             0         
_________________________________________________________________
fc6 (Dense)                  (None, 4096)              73404416  
_________________________________________________________________
dropout_1 (Dropout)          (None, 4096)              0         
_________________________________________________________________
fc7 (Dense)                  (None, 4096)              16781312  
_________________________________________________________________
dropout_2 (Dropout)          (None, 4096)              0         
_________________________________________________________________
fc8 (Dense)                  (None, 1)                 4097      
=================================================================
Total params: 117,845,761
Trainable params: 117,845,761
Non-trainable params: 0
_________________________________________________________________

But this is the result of model.summary() after model = multi_gpu_model(model,2)

Layer (type)                    Output Shape         Param #     Connected to                     
==================================================================================================
conv1_input (InputLayer)        (None, 16, 144, 192, 0                                            
__________________________________________________________________________________________________
lambda_1 (Lambda)               (None, 16, 144, 192, 0           conv1_input[0][0]                
__________________________________________________________________________________________________
lambda_2 (Lambda)               (None, 16, 144, 192, 0           conv1_input[0][0]                
__________________________________________________________________________________________________
sequential_1 (Sequential)       (None, 1)            117845761   lambda_1[0][0]                   
                                                                 lambda_2[0][0]                   
__________________________________________________________________________________________________
fc8 (Concatenate)               (None, 1)            0           sequential_1[1][0]               
                                                                 sequential_1[2][0]               
==================================================================================================
Total params: 117,845,761
Trainable params: 117,845,761
Non-trainable params: 0
__________________________________________________________________________________________________

**Describe the expected behavior**  

**Code to reproduce the issue**  
Provide a reproducible test case that is the bare minimum necessary to generate the problem.  

**Other info / logs**  
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by ziyigogogo** [comment]
If you go to see the source code of the multi_gpu_model you will know the concept.
the Lambda1&Lambda2 is the 2(the #gpu you defined) splits of your input layer. And the sequential_1 wraps the model you defined. This how multi_gpu_model works, nothing to worry about, if you want to print the summary, just print it before multi_gpu_model().

**Comment by raehyuk** [comment]
> If you go to see the source code of the multi_gpu_model you will know the concept.
> the Lambda1&Lambda2 is the 2(the #gpu you defined) splits of your input layer. And the sequential_1 wraps the model you defined. This how multi_gpu_model works, nothing to worry about, if you want to print the summary, just print it before multi_gpu_model().

Thank you! but the problem is when I load the parallel model (model after calling multi_gpu_model), It doesn't work.
